components/library/flatbuffers/flatbuffers/flatbuffers.p5m

```diff
@@ -0,0 +1,72 @@
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2024 Friedrich Kink
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.human-version value=$(HUMAN_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/bin/flatc
+file path=usr/include/flatbuffers/allocator.h
+file path=usr/include/flatbuffers/array.h
+file path=usr/include/flatbuffers/base.h
+file path=usr/include/flatbuffers/buffer.h
+file path=usr/include/flatbuffers/buffer_ref.h
+file path=usr/include/flatbuffers/code_generator.h
+file path=usr/include/flatbuffers/code_generators.h
+file path=usr/include/flatbuffers/default_allocator.h
+file path=usr/include/flatbuffers/detached_buffer.h
+file path=usr/include/flatbuffers/file_manager.h
+file path=usr/include/flatbuffers/flatbuffer_builder.h
+file path=usr/include/flatbuffers/flatbuffers.h
+file path=usr/include/flatbuffers/flatc.h
+file path=usr/include/flatbuffers/flex_flat_util.h
+file path=usr/include/flatbuffers/flexbuffers.h
+file path=usr/include/flatbuffers/grpc.h
+file path=usr/include/flatbuffers/hash.h
+file path=usr/include/flatbuffers/idl.h
+file path=usr/include/flatbuffers/minireflect.h
+file path=usr/include/flatbuffers/pch/flatc_pch.h
+file path=usr/include/flatbuffers/pch/pch.h
+file path=usr/include/flatbuffers/reflection.h
+file path=usr/include/flatbuffers/reflection_generated.h
+file path=usr/include/flatbuffers/registry.h
+file path=usr/include/flatbuffers/stl_emulation.h
+file path=usr/include/flatbuffers/string.h
+file path=usr/include/flatbuffers/struct.h
+file path=usr/include/flatbuffers/table.h
+file path=usr/include/flatbuffers/util.h
+file path=usr/include/flatbuffers/vector.h
+file path=usr/include/flatbuffers/vector_downward.h
+file path=usr/include/flatbuffers/verifier.h
+file path=usr/lib/$(MACH64)/cmake/flatbuffers/BuildFlatBuffers.cmake
+file path=usr/lib/$(MACH64)/cmake/flatbuffers/FlatBuffersSharedTargets-noconfig.cmake
+file path=usr/lib/$(MACH64)/cmake/flatbuffers/FlatBuffersSharedTargets.cmake
+file path=usr/lib/$(MACH64)/cmake/flatbuffers/FlatBuffersTargets-noconfig.cmake
+file path=usr/lib/$(MACH64)/cmake/flatbuffers/FlatBuffersTargets.cmake
+file path=usr/lib/$(MACH64)/cmake/flatbuffers/FlatcTargets-noconfig.cmake
+file path=usr/lib/$(MACH64)/cmake/flatbuffers/FlatcTargets.cmake
+file path=usr/lib/$(MACH64)/cmake/flatbuffers/flatbuffers-config-version.cmake
+file path=usr/lib/$(MACH64)/cmake/flatbuffers/flatbuffers-config.cmake
+file path=usr/lib/$(MACH64)/libflatbuffers.a
+link path=usr/lib/$(MACH64)/libflatbuffers.so \
```

Comment:
Is there a good reason to deliver a static library?
If noit, then please add DROP_STATIC_LIBRARIES= yes to the Makefile and rerun gmake sample-manifest-